.github/workflows/bench.yml

```diff
@@ -8,8 +8,8 @@ env:
   CARGO_TERM_COLOR: always
   RUST_BACKTRACE: 1
   TOOLCHAIN: nightly
-  RUSTFLAGS: -C link-arg=-fuse-ld=lld -C link-arg=-Wl,--no-rosegment
-  PERF_CMD: record -o perf.data -F997 --call-graph lbr -g
+  RUSTFLAGS: -C link-arg=-fuse-ld=lld -C link-arg=-Wl,--no-rosegment, -Cforce-frame-pointers=yes
+  PERF_CMD: record -o perf.data -F997 --call-graph fp -g
```

Comment:
Unrelated?

Reply:
Yes. Still mucking around with trying to find the best flags for `perf`, and don't want to do a whole other parallel CI run for that.